tests/Unit/Plugin/Task/Requests/Extension/RequestsPluginTest.php

```diff
@@ -239,7 +239,7 @@ public static function isSupported()
         );
         $plugin->standardRoutineHandler($event);
 
-        $this->assertEquals(['basic' => '123'], $transport->headers);
+        $this->assertEquals(['Authorization' => ' basic 123'], $transport->headers);
```

Comment:
```suggestion
        $this->assertEquals(['Authorization' => 'basic 123'], $transport->headers);
```
Fix failing unit test.

Reply:
oops